loki/README.md

```diff
@@ -0,0 +1,110 @@
+    ```
+
+3. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    helm repo update
+    ```
+
+### Install the Loki stack
+
+Run the Helm upgrade command, which installs the chart if not present yet.
+ ```bash
+helm upgrade --install --create-namespace -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml --set promtail.enabled=false --set grafana.enabled=false
+```
+
+You can either use the [loki-values.yaml](./loki-values.yaml) provided in this `loki` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
+
+
+### Verify the installation
+
+Check that the `loki` Pod has been created in the Namespace and is in the `Running` state:
+
+```bash
+kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get pod ${HELM_RELEASE_NAME}-0
+```
+
+### Activate log shipment using a LogPipeline
+
+1. Download the [logpipeline](https://raw.githubusercontent.com/kyma-project/examples/main/loki/logpipeline-custom.yaml) and replace the `{release-name}` and `{namespace}` placeholder.
+
+2. Apply the modified LogPipeline:
+
+
+   ```bash
+   kubectl apply -f logpipeline-custom.yaml
+When the status of the applied LogPipeline resource turned into `Running`, the underlying Fluentbit is reconfigured and log shipment to your Loki instance should be active
+> **NOTE:** The used output plugin configuration is using all labels of a Pod to label the Loki log streams. That segregation of the log streams might be not optimal performance wise. Follow [Loki's labelling best practices](https://grafana.com/docs/loki/latest/best-practices/) for a tailormade setup fitting to your workload configuration.
+
+### Verify the setup by accessing logs via the Loki API
+
+1. To access the Loki API, use kubectl port forwarding. Run:
+   ```bash
+   kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME} 3100
+
+### Alternative installation options
+
+The used Helm chart provides configuration for accessing logs with Grafana, which is disabled by default. 
+To enable Grafana, run:
+
+```bash
+helm upgrade --install --create-namespace -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/grafana-values.yaml --set grafana.adminPassword=myPwd --set promtail.enabled=false
+```
+
+To access Grafana, use kubectl port forwarding. Run:
+```bash
+kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME}-grafana 3000:80
+```
+Open Grafana in your browser under http://localhost:3000 and log in with `admin` as password `myPwd`. Password can be easly changed by installation parameter `grafana.adminPassword=myPwd` from command above. 
```

Comment:
```suggestion
## Installation together with Grafana
The used Helm chart supports the deployment of Grafana as well, but is disabled by default. As Grafana provides a very good Loki integration, you might want to install it as well.

To deploy Grafana alongside to Loki and having Loki pre-configured as a datasource, run the following command instead of the original command from the Installation section:

```bash
helm upgrade --install --create-namespace -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/grafana-values.yaml --set grafana.adminPassword=myPwd
```

For accessing the Grafana UI via kubectl port forwarding, run:
```bash
kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME}-grafana 3000:80
```

Open Grafana in your browser under `http://localhost:3000` and log in with user `admin` and the password taken from the above helm command.
```